### ISSUE TITLE
[BUG] Not creating all players from RTF into Config.xml

### ISSUE THREAD

**Issue by danipinus** [Maradonna90/NewGAN-Manager#198]
**Describe the bug**
Some players are not getting faces.

I created the RTF file, and use the NewGan Manager to Generate the Config.xml file

I compared some players of both files, and they are not in Config.xml, that's why some players missing the faces. (even in English language  FM)

How can the Config.xml insert all players in RTF file?

Can you help me pls

Thank you

**Comment by Ayal1** [comment]
Did you use the "Allow Duplicates?" checkbox?

**Comment by danipinus** [comment]
> Did you use the "Allow Duplicates?" checkbox?

No, is that what is for? :O 

Thank you will do it

**Comment by Maradonna90** [comment]
which FM version? On 24 we have following issue #186

**Comment by jharlequin** [comment]
FM24 And this issue continues... I downloaded new 1.4.0 version of newGAN manager and issue still continues. Before that i did r- solution it solves about the players that come with rtf file but main issue is newgan manager doesnt read all rtf file or reads but cant copy them to config file i dont know and i'm very frustrated and i need help

**Comment by Olestol** [comment]
> which FM version? On 24 we have following issue #186

For FM23?

**Comment by jharlequin** [comment]
Someone please help us!!

**Comment by Trust-WoW** [comment]
@jharlequin Can you post your RTF file please or let me know how many lines does the file have or how many MB?

**Comment by jharlequin** [comment]
> @jharlequin Can you post your RTF file please or let me know how many lines does the file have or how many MB?

i tried with smaller rtf files and some players doesnt get any face no matter what happens

**Comment by Trust-WoW** [comment]
Yeah I had the same problem before. How many lines does your RTF file have? Maybe just post it here, I can have a look.
For me the problem was when selecting all players in the view before printing, it didn't work quite right via CTRL+, so not enough players were exported.

**Comment by GAlperen** [comment]
I got this issue, some of the players which I can clearly see in RTF does not appear in the config file. I specificly checked some of the players (some of them are in my team and some of them are not) by their IDs and didn't appear in config file. I tried multiple RTF files with filtering players by their countries then by their positions. I didn't solve it.

**Comment by dorukkazanc** [comment]
Which language are you using? check the config file and if it is matching with the rtf file you created. If you use some language other than ENG it probably creates rtf file with different nation keys

**Comment by iqwik** [comment]
I've got the same issue. The filter (from NewGAN-Manager 1.4.0) works not correct, only return some of players, not all

**Comment by aagobbo** [comment]
I'm using FM22 and i'm in the 45/46 season and my last try was 2036, but had no changes... tried again now, 2045, and still no changes... the rtf is 30.475 KB and 264106  lines and found some players that has no pic... I'm using the 1.3.1 newgan... tried even change to english and nothing... any thoughts?

**Comment by GAlperen** [comment]
I solved my problem, maybe it helps you too.

1. If you are playing in a different language change it to the UK English
2. Start NewGen Manager as admin

These solved my problems. I don't say it will solve yours too. But give it a try.

Edit: You also might want to check data language option, sometimes it don't change even if you change the game language.
